src/app/components/CopyToClipboard/index.tsx

```diff
@@ -51,9 +53,21 @@ export const CopyToClipboard: FC<CopyToClipboardProps> = ({ value }) => {
 
   return (
     <Tooltip arrow onOpen={hideTooltip} open={isCopied} placement="right" title={t('clipboard.success')}>
-      <StyledButton color="inherit" onClick={handleCopyToClipboard} aria-label={ariaLabel}>
-        <ContentCopyIcon sx={{ fontSize: '1.25em', color: COLORS.brandDark }} />
-      </StyledButton>
+      {label ? (
+        <Button
+          variant="outlined"
+          color={'secondary'}
```

Comment:
can we just pass a string instead of jsx expression

Reply:
OK done. (Also cleaned up the same thing everywhere in the code.)